### ISSUE TITLE
Optimization process

### ISSUE THREAD

**Issue by MerceaOtniel** [suragnair/alpha-zero-general#112]
In  the AlphaGo Zero paper they say "Parameters were optimized by stochastic gradient descent with momentum and learning rate annealing, using the same loss as in equation (1), but weighting the MSE component by a factor of 0.01.". In the repo i cannot see where MSE is multiplied by 0.01. Has anyone tried this? There are some improvements in the speed of convergence?

**Comment by alreadydone** [comment]
That's for supervised learning. For training AGZ:

> Neural network parameters are optimized by stochastic gradient descent with momentum and learning rate annealing, using the loss in equation (1). The learning rate is annealed according to the standard schedule in Extended Data Table 3. The momentum parameter is set to 0.9. The cross-entropy and MSE losses are weighted equally (this is reasonable because rewards are unit scaled, r∈{−1, +1}) and the L2 regularization parameter is set to c=10^−4.
